### DISCUSSION TITLE
LCDM cosmological coincidences

### DISCUSSION THREAD

**Discussion by mikehelland** [a-cosmology-group/acg#198] [Cosmology Models]
Here's 1 / Hubble's parameter, H(t), times 1 pc/yr and 1 ly/yr, the dotted lines. And the age and radius also there.

![image](https://github.com/a-cosmology-group/acg/assets/19311107/fbeda0b9-c890-4c5b-9fca-5c1b71e0c088)

I thought it was interesting that the radius of the observable universe, 46 Gly, is 14 Gpc. 

That kind of looks like very fundamental aspects of the universe, line up exactly with the pc / ly conversion rate, right as present time, as opposed to the next few billion or trillion years.

**Comment by HanDeBruijn** [comment]
Let $H=$ Hubble parameter, $G=$ gravitational constant, $\rho_0=$ mass density here and now, OI = from _Origin of Inertia_, the book by Amitabha Ghosh, Mach = another value from the same book, VMT = version of the Variable Mass Theory (and &Lambda;CMD = oh well). Then we have

$$
H = \sqrt{\chi G \rho_0} \quad \mbox{with} \quad \chi = 8\pi \times
\begin{cases}
1/8 & (: \mbox{OI}) \\
1/6 & (: \mbox{Mach}) \\
1/3 & (: \Lambda\mbox{CDM}) \\
0.3883945571 & (: \mbox{VMT})
\end{cases}
$$

But there are other theories about the Hubble parameter saying that

$$
H =
\begin{cases}
2/\mbox{age} & (: \mbox{VMT}) \\
1/\mbox{age} & (: \mbox{other})
\end{cases}
$$

Let us assume that the first set of formulas is correct.
According to the Variable Mass Theory, mass is increasing with time. Consequently, in a Static Euclidean Universe, the mass density must be increasing with time as well. And so the Hubble parameter must be **increasing** with time.
However, according to the second set of formulas, the Hubble parameter is inversely proportional with time and so, because time is always increasing, the Hubble parameter must be **decreasing** with time.
Contradiction. What is it?

**Reply by mikehelland** [reply]
Hi @HanDeBruijn !

You can see from the graph that in LCDM 1/H never actually equals the age of the universe, except sometime relatively close to now, but from there it will diverge forever. 

That's the coincidence. One of them, anyways.

So your second equation isn't actually what LCDM says.

**Reply by HanDeBruijn** [reply]
Indeed. According to (not only) &Lambda;CDM theory we have, with $R=$ radius (of comoving volume), $A=$ (universe) age, $c=$ lightspeed, $H=$ Hubble parameter:

$$
R = \frac{c}{H} \quad \mbox{and} \quad A = \frac{1}{H}
$$

And that would be valid only for that special moment when time from present is precisely **zero**, therefore only here and now? It makes &Lambda;CDM an unbelievable theory, literally.

**Reply by HanDeBruijn** [reply]
[Looking for feedback on metric for the universe, with time dilation but no expansion](https://github.com/orgs/a-cosmology-group/discussions/178#discussioncomment-7411544) has made possible other interesting discoveries.
The formulas in the post as linked can be written as follows, with $t=$ measured time (negative), $\tau=$ Minkowski time, $H=$ Hubble parameter and $p\rightarrow n$ (what's in a name).

$$
H/p.\tau+1=(1+z)^{-1/p}=\left[e^{-Ht}\right]^{-1/p} \quad \mbox{or} \quad 1+H/n.\tau = e^{H/n.t}
$$

One consequence is that time $\tau$ is an approximation for time $t$ which becomes better for larger values of $n$ (why didn't I see before now this beautiful relationship with pure mathematics).

$$
\left(1+\frac{H\tau}{n}\right)^n = e^{H.t} \quad \mbox{with} \quad \lim_{n\to\infty} \left(1+\frac{x}{n}\right)^n = e^x
$$

Another consequence is that there must be a beginning / a moment of creation A(lpha) in the Minkowsy time frame for $\tau = -n/H = -A$ , but **not** in the measured time frame $t$ . The end-result becomes extremely simple.

$$
1 + \tau/A = e^{t/A}
$$

This means that Orbital time $T$ as defined in (my version of) the Variable Mass Theory **is exactly the same as Minkowski time** $\tau$ as defined in Mike Helland's theory, provided that the **Hubble** parameter is based on **age only** according to the above. Many results in my theory can thus be transferred to Mike's theory and vice versa. That's very nice !! To be continued ..

**Reply by mikehelland** [reply]
Just one small detail:

$$R = \frac{c}{H} * 3.261564$$

Where that constant is the conversion rate from parsec to light years. But it's not actually doing the conversion of units.

The comoving distance to the CMB is ~14 Gpc.   
The light travel time distance to the CMB is ~14 Gly.

I thought that was odd. Nothing particularly meaningful. Just a strange coincidence.

**Reply by RedshiftDrift** [reply]
@mikehelland In ΛCDM cosmology it is a coincidence.  The Hubble parameter $H(t)$ varies with time, and the Hubble time $1/H(t)$ is only equal to the age of the universe **today**! See e.g. [https://physics.stackexchange.com/q/160857](https://physics.stackexchange.com/q/160857)

It's another one of several coincidences that makes an expanding model of space untenable.

**Reply by HanDeBruijn** [reply]
Let us resume with the formula we ended with; let $A=$ age, $t=$ atomic time, $\tau=$ orbital time.

$$
1+\tau/A = e^{t/A} \quad \Longleftrightarrow \quad t = A\ln(1+\tau/A)
$$

Now calculate, for example, velocities $v$ and accelerations $a$ from displacements $s$ within the two time frames.

$$
\vec{v}_\tau = \frac{d\vec{s}}{d\tau} = \frac{d\vec{s}}{dt}\frac{dt}{d\tau} = \vec{v}_t\frac{1}{1+\tau/A}
$$

$$
(1+\tau/A)\vec{v}_\tau = \vec{v}_t
$$

Patiently differentiating further with product rule and chain rule gives

$$
\vec{a}_\tau = \frac{d^2\vec{s}}{d\tau^2} = \frac{d}{d\tau}\left(\frac{d\vec{s}}{dt}\frac{dt}{d\tau}\right) = \frac{d}{d\tau}\left(\frac{d\vec{s}}{dt}\right)\frac{dt}{d\tau} + \frac{d\vec{s}}{dt}\frac{d}{d\tau}\left(\frac{dt}{d\tau}\right) =
$$

$$
= \frac{d^2\vec{s}}{dt^2}\left(\frac{dt}{d\tau}\right)^2 + \frac{d\vec{s}}{dt}\frac{d^2t}{d\tau^2}
$$

Hence

$$
\vec{a}_\tau = \vec{a}_t\left(\frac{1}{1+\tau/A}\right)^2 + \vec{v}_t\frac{-1/A}{(1+\tau/A)^2}
$$

$$
(1+\tau/A)^2 \vec{a}_\tau = \vec{a}_t - \vec{v}_t/A
$$

Under **eartly circumstances** here and now we have that $\tau=0$ and so

$$
\vec{v}_\tau = \vec{v}_t
$$

$$
\vec{a}_\tau = \vec{a}_t - \vec{v}_t/A
$$

Conclusion:  **velocities** in the two time frames are the **same**, but **accelerations** are **not**.

### Application

A quote from the book _Origin of Inertia_ by by Amitabha Ghosh: _It is now an established fact that the spin motion of the Earth is gradually slowing down, and the magnitude of this secular retardation is about_ $6 \times 10^{-22}\mbox{ } rad/s^2$ . That amount of information is already sufficient. According to our theory we have - as repeated from the above:

$$
\vec{a}_\tau = \vec{a}_t - \vec{v}_t/A
$$

Here $\vec{a}_\tau$ is the "true" Newtonian acceleration (namely in orbital time), $\vec{a}_t$  is the acceleration as measured nowadays with atomic clocks and $A$ is an age. All accelerations are proportional with $R=$ radius of the earth. If there is a discrepancy $\Delta\omega$ between the orbital and atomic angular velocities, then it must being caused by the negative term, which is a vector in tangential direction slowing down the rotation. The calculation is a one-liner.

$$
-|\vec{v}_t |/A = -\omega R/A \quad \Longrightarrow \quad \dot{\omega} = -\left(\frac{2\pi}{\mbox{day}}\right)/\mbox{age}
$$

```
# Age of the earth
age := 4.543*10^9;
# Seconds in a year
year := 31556926;
# Seconds in a day
day := 60*60*24;
# Rotation of the earth
omega := 2*Pi/day;
# Retardation
slower := evalf(omega/(age*year));
```
We find that the retardation is about $5 \times 10^{-22} \mbox{ } rad/s^2$

**Reply by HanDeBruijn** [reply]
@mikehelland .
>Just one small detail:

$$
R = \frac{c}{H} * 3.261564
$$

A slightly different value (with underpinning) is presented in the [G. de Vaucouleurs](http://www.alternatievewiskunde.nl/cosmology/helaas.htm) section at my website:

$$
R = \frac{c}{H} * 3.383634283
$$

Also take notice of the fact that I'm no longer saying that $H = 1/A$ or $H = 2/A$ but instead that _the Hubble parameter is  inversely proportional to **A**ge_, with an arbitrary factor $n$, not even necessarily an integer: $H = n/A$ .

**Reply by HanDeBruijn** [reply]
We repeat.
Let $H=$ Hubble parameter, $\chi=$ algebraic constant, $G=$ gravitational constant, $\rho_0=$ mass density here and now, then we have

$$
H = \sqrt{\chi G \rho_0}
$$

But there are other theories saying that the Hubble parameter is in general inversely proportional to **A**ge:

$$
H = \frac{n}{A} \quad \mbox{with} \quad 
\begin{cases}
n=1 & (: \mbox{TDP}) \\
n=2 & (: \mbox{VMT}) \\
n = 3.383634283 & (: \mbox{UAC})
\end{cases}
$$

Here TDP = [Time Dilated Past](https://github.com/orgs/a-cosmology-group/discussions/160), VMT = [Variable Mass Theory](https://www.youtube.com/watch?v=T7jVviVBzKs), UAC = [Unified Alternative Cosmology](http://www.alternatievewiskunde.nl/cosmology/wild.htm).
According to the last set of formulas we have, dependent on the point of observation $(1,2)$ in space and time:

$$
\frac{H_1}{H_2} = \frac{A_2}{A_1}
$$

The other way around is a bit more tricky.

$$
\frac{H_1}{H_2} = \frac{\sqrt{\chi G \rho_1}}{\sqrt{\chi G \rho_2}} = \frac{\sqrt{\rho_1}}{\sqrt{\rho_2}}
$$

The mass density $\rho$ is _mass divided by expanding empty space_. According to previous theory we have, for mass $M$, when integrated over a whole (part of the) universe

$$
M_1 = \rho_1 \int_0^R e^{-H/c.r}.4\pi r^2.dr \quad ; \quad M_2 = \rho_2 \int_0^R e^{-H/c.r}.4\pi r^2.dr
$$

However, when talking about densities $(\rho_1,\rho_2)$ **here and now**, the above volume $[0,R]$ is far to large.
Sober thinking reveals that the calculations should be replaced by the following, over a very limited range $dr$ (i.e _infinitesimal_ when taken on a cosmological scale). And the masses $m$ are much smaller.

$$
m_1 = \rho_1 . 4\pi r_1^2.dr \quad ; \quad m_2 = \rho_2 . 4\pi r_2^2.dr
$$

$$
\rho_1 = \frac{m_1}{4\pi r_1^2.dr} \quad ; \quad \rho_2 = \frac{m_2}{4\pi r_2^2.dr}
$$

So now we have, with length contraction equivalent to the expansion of empty space, that is with $r_1 \sim m_1$ and $r_2 \sim m_2$ :

$$
\frac{H_1}{H_2} = \frac{\sqrt{\rho_1}}{\sqrt{\rho_2}} = \sqrt{\frac{m_1/(m_1^2)}{m_2/(m_2^2)}} =
\sqrt{\frac{m_2 \sim A_2^2}{m_1 \sim A_1^2}} = \frac{A_2}{A_1}
$$

Great relief !! this is **consistent with** the above interpretation of the Hubble parameter as the inverse of an **age**.

**Reply by mikehelland** [reply]
H = n/A is, no pun intended, not applicable to any theory I know of.

LCDM does not, nor does any FLRW model (on which LCDM is based) say anything like that.

What they actually say is shown in the graph above. 

Calculate H(t) at t=-4 billion and t=+4 billion years using LCDM. You'll see H and A have nothing in common. It's just this weird time in cosmic history relatively close to now where they are similar.

**Reply by HanDeBruijn** [reply]
@mikehelland .
>H = n/A is [ .. ] not applicable to any theory I know of.

Oh well, this has been revealed in several threads that were steered by you, for example in the above [reply](https://github.com/orgs/a-cosmology-group/discussions/198#discussioncomment-8303773).
What's even more amazing to me, according to your own [Time Dilated Past hypothesis](https://github.com/orgs/a-cosmology-group/discussions/160) we have the following formula.

$$
1+H\tau = e^{Ht}
$$

This means that your universe is eternal in (previously called) measured time $t$ and has a beginning in that so-called Minkowski time $\tau$. The reason is that, for $t \to - \infty$ :

$$
1+H\tau = e^{-\infty} = 0 \quad \Longleftrightarrow \quad \tau = -1/H = -A
$$

With other words, in **your own** theory we have an **A**ge that is connected to the **H**ubble parameter.

$$
H=n/A \quad \mbox{with} \quad n = 1
$$

>You'll see H and A have nothing in common.

Sorry?
>It's just this weird time in cosmic history relatively close to now where they are similar.

Maybe it's the case in &Lambda;CDM, but that theory is rejected by many, if not the most [of us](https://github.com/orgs/a-cosmology-group/discussions/198#discussioncomment-8306628) here. It's anyway impossible in an eternal and infinite Static Euclidean Universe that such a "coincidence" would be restrained to _this weird time in cosmic history relatively close to_ here and now, because **observers at _any_ place and _any_ time are equivalent** in such a cosmos.

**Reply by mikehelland** [reply]
> Maybe it's the case in ΛCDM, but that theory is rejected by many

Sure. The point of the thread was just to talk about some of the coincidences in that theory.

FWIW, I (personally) would not claim that the universe is infinite or eternal. One reason is those are bold claims and there's no empirical evidence of that. Another is that my model actually says nothing about the age of the universe. Here is what Edwin Hubble said:

>The familiar interpretation of red-shifts as velocity-shifts very seriously restricts not only the time scale, the age of the universe, but the spatial dimensions as well. On the other hand, the alternative possible interpretation, that red-shifts are not velocity-shifts, avoids both difficulties, and presents the observable region as an insignificant sample of a universe that extends indefinitely in space and in time. 

That's how I see it. The universe is **indefinitely** old and large. That's more scientific to my ears than _eternal and infinite_.

**Reply by HanDeBruijn** [reply]
>Another is that my model actually says nothing about the age of the universe.

Perhaps not explicitly, but implicitly .. ?? **Please answer** this question: **what is the name** of the value $A=1/H$ of Minkowski time that is equivalent to $t\to -\infty$ as in

$$
1+H\tau = e^{-\infty} = 0 \quad \Longleftrightarrow \quad \tau = -1/H = -A
$$

>That's how I see it. The universe is **indefinitely** old and large. That's more scientific [ .. ]

I could not agree more, Mike !! _What's in a word_ is the summary of our misunderstanding, it seems. Here is what Gauss said:

>[ Was nun aber Ihren Beweis f&uuml;r (1) betrifft, ] 
so protestire ich zuv&ouml;rderst gegen den Gebrauch einer unendlichen Gr&ouml;&szlig;e
als einer Vollendeten, welcher in der Mathematik niemals erlaubt ist. Das
Unendliche ist nur eine fa&ccedil;on de parler, indem man eigentlich von Grenzen
spricht, denen gewisse Verh&auml;ltnisse so nahe kommen als man will, w&auml;hrend
andern ohne Einschr&auml;nkung zu wachsen verstattet ist.

: [C.F. Gauss](http://en.wikipedia.org/wiki/Carl_Friedrich_Gauss) in a letter to Schumacher, 12 July 1831. Translation:

>I protest against the use of infinite magnitude as something completed, which
is never permissible in mathematics. **Infinity is merely a way of speaking**, the
true meaning being a limit which certain ratios approach **indefinitely** close,
while others are permitted to increase **without restriction**.

**Reply by mikehelland** [reply]
I would call 1/H the Hubble limit, as that's it's name. At that point, time is dilated to infinity, and all information before that is lost.

**Reply by HanDeBruijn** [reply]
>At that point, time is dilated to infinity

More precise: "measured" time $t$ is dilated to infinity.
But there is also time $\tau$ in the Minskowski frame which is _not_ dilated to infinity.
>and all information before that is lost.

Information? Before that? **I am** lost here.

**Reply by mikehelland** [reply]
Minkowski time isn't a real thing, it's what you'd have, hypothetically, without time dilation.

The idea is that the universe goes back in time indefinitely, it's just everything before 14 billion years ago is on the wrong side of the Hubble limit, meaning it is in accessible to us.
